scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -735,6 +735,7 @@ <h4 class="modal-title" id="filterNameModalLabel">Filter display name</h4>
     <input type="submit" name="page" id="paginate-first" value=1 hidden="true">
     <input type="submit" name="page" id="paginate-previous" value={{page-1}} hidden="true">
     <input type="submit" name="page" id="paginate-next" value={{page+1}} hidden="true">
+    <input type="hidden" name="page" value="{{page}}"> <!-- In order to work, this input should be placed AFTER the submit form items with the same name -->
```

Comment:
😮 I didn't know that! Nice solution! Works fine in Safari; is it the same for all browsers?

Reply:
I think so. The explanations can be found in the HTML rules: https://html.spec.whatwg.org/multipage/forms.html#configuring-a-form-to-communicate-with-a-server

It says: `If there are multiple matching items, then a RadioNodeList object containing all those elements is returned.` which means a list. And the list is populated according to the occurrence of the items. 

When no submit button is pressed the the list has one element (the hidden field). If you press a button it will have 2 elements (the button and the hidden div). 

If you want to see both elements of this list you should do something like form.page.getList, but since in the controller we use only form.get("page") it returns only the first element. 

Perhaps not the clearest way of make it work, but looks like it does!

Reply:
Are still satisfied with the solution after knowing how it works ? 😆

Reply:
Certainly; we have only had a little bit of complaints about this so far, and if this works for at least most browsers it's all good.